src/tools/clippy/tests/ui/char_indices_as_byte_indices.rs

```diff
@@ -1,4 +1,3 @@
-#![feature(round_char_boundary)]
```

Comment:
You need to run `./x.py test clippy --bless` to update the `stderr` file with the line number changes.